cognite/v04/dto.py

```diff
@@ -212,7 +213,7 @@ def to_json(self):
 
     def to_pandas(self):
         '''Returns data as a pandas dataframe'''
-        items = self.internal_representation['data']['items']
+        items = deepcopy(self.internal_representation['data']['items'])
```

Comment:
Does it make sense to write a test that proves that the mutation bug is fixed?

Reply:
Good idea - makes perfect sense.